src/types/zetkin.ts

```diff
@@ -4,6 +4,7 @@ export interface ZetkinCampaign {
     title: string;
     id: number;
     organization?: ZetkinOrganization;
+    manager: string | null;
```

Comment:
I'm pretty sure this wouldn't be a `string`, but an object like below, but we can fix that once it actually exists.

```ts
manage: null | {
  id: number,
  name: string,
}
```